ogn/parser/pattern.py

```diff
@@ -52,19 +52,18 @@
     )?
 """, re.VERBOSE | re.MULTILINE)
 
-
 PATTERN_AIRCRAFT_BEACON = re.compile(r"""
```

Comment:
@svoop: Could you have a look at this regex? I changed it because the aircraft beacon can now just include details, id, climb_rate and turn_rate.